src/ast/descriptor.ts

```diff
@@ -0,0 +1,8 @@
+import * as jsdoc from '../ast-utils/jsdoc';
+
+export type LiteralValue = string|number|boolean|RegExp;
+
+export interface Descriptor {
```

Comment:
Punt for now, we'll be forced to figure this out when the first clients come online.